zk/witness/witness.go

```diff
@@ -362,3 +368,16 @@ func getWitnessBytes(witness *trie.Witness, debug bool) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func (g *Generator) generateMockWitness(batchNum uint64, blocks []*eritypes.Block, debug bool) ([]byte, error) {
+	mockWitness := []byte("mockWitness")
+
+	if debug {
+		log.Info("Generated mock witness", "batch", batchNum, "witness", mockWitness)
+		for i, block := range blocks {
+			log.Info("Block", "number", i, "number", block.NumberU64(), "hash", block.Hash().Hex())
```

Comment:
This could get a little noisy in the logs, empty batches have around 120 blocks in them.  I think just the start and end block would be useful in the top level log along with the batch number.

Reply:
I agree, logging just start, end block, batch num and witness